mhcflurry/class1_affinity_prediction/class1_affinity_predictor.py

```diff
@@ -119,6 +119,27 @@ def supported_peptide_lengths(self):
             max(lower for (lower, upper) in length_ranges),
             min(upper for (lower, upper) in length_ranges))
 
+    def merge(self, affinity_predictors=[]):
+        """ 
+        Merge multiple affinity predictors into one.  
```

Comment:
Probably want to say you're merging multiple affinity predictors into the *current predictor*